Makefile

```diff
@@ -162,21 +160,14 @@ vet:
 fmt:
 	find . -type f -name "*.go" | grep -v "./vendor/*" | xargs gofmt -s -w -l
 
-# Run the bootstrap target once before trying gometalinter in Development environment
-.PHONY: golint
-golint:
-	@gometalinter.v1 --install
-	@gometalinter.v1 --vendor --deadline=600s ./...
-
 # shellcheck target for checking shell scripts linting
 .PHONY: shellcheck
 shellcheck: getshellcheck
 	find . -type f -name "*.sh" | grep -v "./vendor/*" | xargs /tmp/shellcheck-stable/shellcheck
 
 .PHONY: getshellcheck
 getshellcheck:
-	wget -c 'https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz' --no-check-certificate -O - | tar -xvJ -C /tmp/
-
+ 	wget -c 'https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz' --no-check-certificate -O - | tar -xvJ -C /tmp/
```

Comment:
This is not required right? The change is already present.

Reply:
The previous build failed for shell check, I assumed I forgot to add the URL fix but even now it fails. I can git reset and hard push once we figure out what's causing it

Reply:
The error says not able to find the shellcheck binary in the path

Reply:
It finds it in other architectures.

Reply:
shellcheck is enabled only in `amd64`, thats why its not failing in other architectures.

Reply:
Oh. How do we fix it?